java/core/src/test/java/com/microsoft/bond/testmodel/A.java

```diff
@@ -285,6 +284,11 @@ protected final void deserializeStructFields(TaggedDeserializationContext contex
         }
 
         @Override
+        protected final void deserializeStructFields(UntaggedDeserializationContext context, A<X, Y> value) throws IOException {
+        }
```

Comment:
Not sure what you mean. I don't want to maintain these files by hand now that we have golden files set up.